features/password_reset.feature

```diff
@@ -46,3 +46,16 @@ Feature: Password Reset
     And I press "Email me a link to reset my password"
     Then I should see "No user was found with that email address"
     And I should be on the password reset page
+
+  Scenario: Trying to reset password to the password currently in the database
```

Comment:
As cucumber tests are meant to echo the actual user experience, rather than anything going on under the hood, I would rename this scenario to something less technical, such as `Trying to re-use the same password`.